lib/ansible/cli/doc.py

```diff
@@ -1046,12 +1046,7 @@ def namespace_from_plugin_filepath(filepath, plugin_name, basedir):
     @staticmethod
     def _combine_plugin_doc(plugin, plugin_type, doc, plainexamples, returndocs, metadata):
         # generate extra data
-        if plugin_type == 'module':
```

Comment:
i would keep plugin_type check as it is a lot less expensive and will false positives in  `plugin in action_loader`